debugging-w-chatgpt/debugging-w-chatgpt.md

```diff
@@ -325,27 +325,26 @@ Once we’ve found and fixed the bug, we need to test our code to make sure the
 
 As stated before, we can’t wholeheartedly trust everything ChatGPT says. Evaluating a code response from ChatGPT should look pretty similar to evaluating any other information created by a generative AI - we need to look for verifiable pieces of information in the response and use other trustworthy sources to confirm if those pieces of information are true. Check out the lesson “Prompting Basics & Using ChatGPT - Verifying Information from ChatGPT” if you’d like to refresh yourself on strategies for verifying information.
 
-Let’s say that a student, Althea, is working on [the Adagrams project](https://github.com/AdaGold/adagrams-py) and is debugging an `IndexError` that sometimes appears in their `draw_letters` function:
+Let’s say that a student, Althea, is working on a game with a function that draws 7 random domino tiles from a pool. She is debugging an `IndexError` that sometimes appears in their `draw_tiles` function:
 
 ```
-def draw_letters():
-    distribution = {'A': 9, 'B': 2, 'O': 8, 'C': 2, 'P': 2, 
-                    'D': 4, 'Q': 1, 'E': 12, 'R': 6, 'F': 2, 
-                    'S': 4, 'G': 3, 'T': 6, 'H': 2, 'U': 4, 
-                    'I': 9, 'V': 2, 'J': 1, 'W': 2, 'K': 1, 
-                    'X': 1, 'L': 4, 'Y': 2, 'M': 2, 'Z': 1}
-    pool = []
-    for letter in distribution:
-        pool.extend([letter for _ in range(distribution[letter])])
-
-    hand = []
-    for _ in range(10):
+def draw_tiles():
+    HAND_SIZE = 7
+    pool = [
```

Comment:
I could also make `pool` a list of lists too:
```
pool = [
    [0,0], [0,1]
]
```

Reply:
I like it as-is using the strings. I think if someone were to play with the example and print the hand, it'd look more like they expect without taking extra steps.